### DISCUSSION TITLE
Does not work

### DISCUSSION THREAD

**Discussion by warbe-maker** [allejo/jekyll-toc#49] [Q&A] [answered]
It's obviously my limited knowledge which caused I failed in getting the toc shows up. Can you give an example what the {{ content }} will look with liquid toc? My last try looks as follows:
```
 <div class="post-content e-content" itemprop="articleBody">
    {%- unless page.notoc -%}
    {%- assign my_min = page.toc_min | default: site.toc_min | default: 2 -%}
    {%- assign my_max = page.toc_max | default: site.toc_max | default: 3 -%} 
    {%- assign my_name = page.url | default: "unnamed" -%}
    <div id="side-toc-title">On this page:</div> 
    {%- include toc_pure_liquid.html html=content sanitize=true class="inline_toc" id="my_toc" toc_min=my_min toc_max=my_max page_name=my_name -%}
    {%- endunless -%}
    {{content}}
  </div>
```
Any help would be very much appreciated.

**Comment by allejo** [comment]
The first thing would be to make sure you're using the latest [`toc.html`](https://github.com/allejo/jekyll-toc/blob/master/_includes/toc.html); the original `toc_pure_liquid.html` filename is the old version that was released over 3 years ago.

Do you have an example repository I can take a look at? I don't see anything glaringly wrong with your snippet so it might be that `content` doesn't exist wherever you're using it.

**Comment by warbe-maker** [comment]
Many thanks for your fasr reply,
For the  toc.html I've copied the content from yours into my toc.html in the _includes folder. 
The snippet I've copied from one of the sources mentioned as demonsration. First I thought I will see (get demonstrated) how the toc looks on the site. When I landed in the repo I thought, well, maybe it's the way this toc is incorporated in the post.html. Unfortunately the demonstration was an old one and a page.html. Anyhow, I modified it and now it is as follows:
```
<div class="post-content e-content" itemprop="articleBody">
   {%- unless post.notoc -%}
	{%- assign my_min = post.toc_min | default: site.toc_min | default: 2 -%}
	{%- assign my_max = post.toc_max | default: site.toc_max | default: 3 -%} 
	{%- assign my_name = post.url | default: "unnamed" -%}
	<div id="side-toc-title">In this post:</div> 
	{%- include toc.html html=content sanitize=true class="inline_toc" id="my_toc" toc_min=my_min toc_max=my_max page_name=my_name -%}
    {%- endunless -%}
    {{content}}
</div>
```
I was quite optimistic - but it  failed.
You may have a look at: [my blog repo](https://github.com/warbe-maker/warbe-maker.github.io) or the result of it [Excel Passion](https://warbe-maker.github.io).

Please have a look. Without your help I don't see any chance to succeed. My knowledge is too limited. Proved when I thought I got it.

**Comment by warbe-maker** [comment]
I've checked all posts - and it worked in all the others. In one I will use notoc: true in the front matter which (I hope) will suppress the empty toc. Why the toc does not show up in the one I've always looked at is definitely my problem because your liquid toc works!!
Thanks a lot for your help.

**Comment by warbe-maker** [comment]
Everyting return! What I saw was my own manually maintained tocs. The only thing I do see as a sign from your liquid toc is the title which I've chosen "In this post:" (while I use "In this post" for the manually maintained tocs to distinguish them from "yours" (once the show up). Sorry for the back and forth.

**Comment by allejo** [comment, marked as answer]
I've sent you a PR over at https://github.com/warbe-maker/warbe-maker.github.io/pull/2, things are working fine for me
